frappe/utils/boilerplate.py

```diff
@@ -127,12 +127,12 @@ def make_boilerplate(dest, app_name):
 # ------------------
 
 # include js, css files in header of desk.html
-# app_include_css = "/assets/{app_name}/css/{app_name}.css"
-# app_include_js = "/assets/{app_name}/js/{app_name}.js"
+# app_include_css = "/assets/css/{app_name}.css"
```

Comment:
on `bench build` it creates symlinks

On Wed, May 4, 2016 at 6:19 PM, Valmik <notifications@github.com> wrote:

> In frappe/utils/boilerplate.py
> <https://github.com/frappe/frappe/pull/1739#discussion_r62032283>:
>
> > @@ -127,12 +127,12 @@ def make_boilerplate(dest, app_name):
> >  # ------------------
> >
> >  # include js, css files in header of desk.html
> > -# app_include_css = "/assets/{app_name}/css/{app_name}.css"
> > -# app_include_js = "/assets/{app_name}/js/{app_name}.js"
> > +# app_include_css = "/assets/css/{app_name}.css"
>
> Was working on Schools with @neilLasrado <https://github.com/neilLasrado>
> , noticed that it wasn't pulling from that dir.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/frappe/frappe/pull/1739/files/751be69757db0ed5a5c05d61c48b19a27cc1d3bb#r62032283>
>